api/src/test/java/org/pmiops/workbench/zendesk/ZendeskRequestsTest.java

```diff
@@ -0,0 +1,115 @@
+package org.pmiops.workbench.zendesk;
+
+import static com.google.common.truth.Truth.assertThat;
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Before;
+import org.junit.Test;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.ResearchPurpose;
+import org.pmiops.workbench.model.SpecificPopulationEnum;
+import org.pmiops.workbench.model.Workspace;
+
+public class ZendeskRequestsTest {
+  private DbUser user;
+
+  @Before
+  public void setUp() {
+    user = new DbUser();
+    user.setGivenName("Falco");
+    user.setFamilyName("Lombardi");
+    user.setUsername("flaco@fake-research-aou.org");
```

Comment:
Typo here. Seems like it would be difficult to type with one's wings.

Reply:
Hah - this was actually an intentional typo, but I've fixed it here to avoid confusion.